main.nf

```diff
@@ -21,78 +21,66 @@ version = '0.1'
 
 // Configurable variables - same as NGI-RNAseq for now
 params.project = false
-params.genome = false
-params.forward_stranded = false
-params.reverse_stranded = false
-params.unstranded = false
-params.star_index = params.genome ? params.genomes[ params.genome ].star ?: false : false
-params.fasta = params.genome ? params.genomes[ params.genome ].fasta ?: false : false
-params.gtf = params.genome ? params.genomes[ params.genome ].gtf ?: false : false
-params.bed12 = params.genome ? params.genomes[ params.genome ].bed12 ?: false : false
-params.hisat2_index = params.genome ? params.genomes[ params.genome ].hisat2 ?: false : false
-params.multiqc_config = "$baseDir/conf/multiqc_config.yaml"
-params.splicesites = false
-params.download_hisat2index = false
-params.download_fasta = false
-params.download_gtf = false
-params.hisatBuildMemory = 200 // Required amount of memory in GB to build HISAT2 index with splice sites
-params.saveReference = false
-params.saveTrimmed = false
-params.saveAlignedIntermediates = false
 params.reads = "data/*{1,2}.fastq.gz"
 params.outdir = './results'
 params.email = false
-params.STAR = false
-params.FUSIONCATCHER = true
+params.star = false
+params.fusioncatcher = true
 Channel
-    .fromFilePairs( params.reads, size:  2 )
+    .fromFilePairs( params.reads, size: 2 )
     .ifEmpty { exit 1, "Cannot find any reads matching: ${params.reads}" }
-    .into { read_files_STAR-fusion; fusionInspector-reads; fusioncatcer-reads}
-
+    .into { read_files_star_fusion; fusion_inspector_reads; fusioncatcher_reads}
 
 
 /*
- * STEP 1 - STAR-Fusion 
+ * STEP 1 - STAR-Fusion
  */
+process star_fusion{
 
-process STAR_fusion{
-        
     input:
-    set val (name), file(read1), file(read2) from read_files_STAR-fusion   
+    set val (name), file(read1), file(read2) from read_files_star_fusion
 
     output:
-    '*final.abridged*'
-    'star-fusion.fusion_candidates.final.abridged.FFPM' into fusion_candidates
-    
-    when: STAR == true 
-    
+    file '*final.abridged*' into star_fusion_abridged
+    file 'star-fusion.fusion_candidates.final.abridged.FFPM' into fusion_candidates
+
+    when: params.star
+
+    script:
     """
-    STAR-Fusion --genome_lib_dir ${star_fusion_refrence} -_left_fq ${read1} --right_fq ${read2}  --output_dir ${star-fusion_outdir}
+    STAR-Fusion \
+        --genome_lib_dir $star_fusion_refrence \
+        --left_fq $read1 \
+        --right_fq $read2 \
+        --output_dir $name
     """
 }
 
 
 /*
  *  -  FusionInspector
  */
-
-process FusionInspector{
+process fusioninspector {
 
     input:
-    set val (name), file(read1), file(read2) from fusionInspector-reads 
-    file fusion_candidates 
-    
+    set val (name), file(read1), file(read2) from fusion_inspector_reads
+    file fusion_candidates
+
     output:
+    file '*' into fusioninspector_results
```

Comment:
You had `output:` here with nothing in the section, which was probably confusing the hell out of Nextflow..